src/ReactiveUI/ObservableForProperty/ObservableAsPropertyHelper.cs

```diff
@@ -98,6 +98,44 @@ public sealed class ObservableAsPropertyHelper<T> : IHandleObservableErrors, IDi
             T initialValue = default,
             bool deferSubscription = false,
             IScheduler? scheduler = null)
+            : this(observable, onChanged, onChanging, () => initialValue, deferSubscription, scheduler)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ObservableAsPropertyHelper{T}"/> class.
+        /// </summary>
+        /// <param name="observable">
+        /// The Observable to base the property on.
+        /// </param>
+        /// <param name="onChanged">
+        /// The action to take when the property changes, typically this will call
+        /// the ViewModel's RaisePropertyChanged method.
+        /// </param>
+        /// <param name="onChanging">
+        /// The action to take when the property changes, typically this will call
+        /// the ViewModel's RaisePropertyChanging method.
+        /// </param>
+        /// <param name="getInitialValue">
+        /// The function used to retrieve the initial value of the property.
+        /// </param>
+        /// <param name="deferSubscription">
+        /// A value indicating whether the <see cref="ObservableAsPropertyHelper{T}"/>
+        /// should defer the subscription to the <paramref name="observable"/> source
+        /// until the first call to <see cref="Value"/>, or if it should immediately
+        /// subscribe to the <paramref name="observable"/> source.
+        /// </param>
+        /// <param name="scheduler">
+        /// The scheduler that the notifications will provided on - this
+        /// should normally be a Dispatcher-based scheduler.
+        /// </param>
+        public ObservableAsPropertyHelper(
+            IObservable<T> observable,
+            Action<T> onChanged,
+            Action<T>? onChanging = null,
+            Func<T>? getInitialValue = null,
```

Comment:
I have to declare an optional parameter to avoid having an overload with the `onChanging` parameter in another position.